ui/src/common/components/LastUpdateDate.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment-timezone';
+import { LOCAL_TIMEZONE } from '../constants';
+
+class LastUpdateDate extends Component {
```

Comment:
although nothing is wrong here, we prefer to write new components as functions.

let's name it `RecordUpdateInfo`